src/spaces/vectorspaces.jl

```diff
@@ -67,11 +68,14 @@ function dim end
 """
     dual(V::VectorSpace) -> VectorSpace
 
-Return the dual space of `V`; also obtained via `V'`. It is assumed that
-`typeof(V) == typeof(V')`.
+Return the dual space of `V`; also obtained via `V'`. This should satisfy 
+dual(dual(V))==V. It is assumed that `typeof(V) == typeof(V')`.
```

Comment:
Could you make this
```
`dual(dual(V)) == V`
```
 i.e. with backticks and spaces. Then it prints more nicely as code. Thanks.

Reply:
Done!